acquire/acquire.py

```diff
@@ -2269,6 +2278,20 @@ def main() -> None:
     log.info("Default Arguments: %s", " ".join(args.config.get("arguments")))
     log.info("")
 
+    # start GUI if requested through CLI / config
+    flavour = "cli"
```

Comment:
```suggestion
    flavour = None
```
I don't think setting this to `cli` actually does anything currently? May as well make it `None` then.

Reply:
Actually I see there's an explicit check for a `None` flavour in the `__new__`. Why is that? Is that actually necessary? It's not documented or commented why it would be a logic flaw.